docs/content/integrations/airflow/migrating-to-dagster.mdx

```diff
@@ -55,19 +55,31 @@ Even in the case that requires some infrastructure investment, `dagster-airflow`
 
 ---
 
-## Limitations
+## Deciding on persistent vs ephemeral airflow database
 
-Before kicking off a migration, note that there are some limitations to converting Airflow DAGs into Dagster jobs, assets, and schedules. The following features are currently unsupported:
+The dagster airflow migration tooling supports two methods for persisting airflow metadatabase state. By default it will us an ephemeral database that is scoped to every job run and thrown away as soon a job run terminates. You can also configure a persistent database that will be shared by all job runs. This tutorial uses the ephemeral database but you will want to use the persistent database option if you need the following features:
```

Comment:
```suggestion
The Dagster Airflow migration tooling supports two methods for persisting Airflow metadatabase state. By default, it will use an ephemeral database that is scoped to every job run and thrown away as soon as a job run terminates. You can also configure a persistent database that will be shared by all job runs. This tutorial uses the ephemeral database, but the persistent database option is recommended if you need the following features:
```